librz/core/project_migrate.c

```diff
@@ -500,6 +475,62 @@ RZ_API bool rz_project_migrate_v11_v12(RzProject *prj, RzSerializeResultInfo *re
 	return true;
 }
 
+// --
+// Migration 12 -> 13
+//
+// Changes from 5badfe10f655374f424164d62915cb3a1d22e2b9:
+//	Removed typelinks from "/core/analysis/typelinks"
+//	and converted into global variables in "/core/analysis/vars"
+
+typedef struct {
+	RzList /*<char *>*/ *moved_keys; ///< deferred for deletion from the old sdb
+	Sdb *global_vars_db;
+} V3V4TypesCtx;
```

Comment:
```suggestion
} V12V13TypesCtx;
```

and other namings below